packages/libs/deer/src/lib.rs

```diff
@@ -44,23 +44,61 @@ mod schema;
 extern crate alloc;
 
 pub trait ObjectAccess<'de> {
+    /// This enables bound-checking for [`ObjectAccess`]. After calling this [`ObjectAccess`] will
```

Comment:
rustdoc uses the first paragraph as summary on the parent page. This needs reformatting.
```suggestion
    /// This enables bound-checking for [`ObjectAccess`].
    ///
    /// After calling this [`ObjectAccess`] will
```